### PR TITLE
chore: just a little bit of cleanup

### PR DESCRIPTION
**Pull request by paperclover**

### CODE REVIEW

File: src/shell/interpreter.zig [hunk 1]
```diff
@@ -3278,7 +3278,7 @@ pub fn NewInterpreter(comptime EventLoopKind: JSC.EventLoopKind) type {
                         const err = switch (this.state) {
                             .expanding_redirect => this.state.expanding_redirect.expansion.state.err,
                             .expanding_args => this.state.expanding_args.expansion.state.err,
-                            else => @panic("Invalid state"),
+                            else => |t| std.debug.panic("Unexpected state .{s} in Bun shell", .{@tagName(t)}),
```
Comment (Jarred-Sumner) [inline src/shell/interpreter.zig:3281]: Output.panic, we should ban std.debug.panic

File: src/shell/interpreter.zig [hunk 2]
```diff
@@ -6732,7 +6732,7 @@ pub fn NewInterpreter(comptime EventLoopKind: JSC.EventLoopKind) type {
                 pub fn onAsyncTaskDone(this: *Rm, task: *ShellRmTask) void {
                     var exec = &this.state.exec;
                     const tasks_done = switch (exec.state) {
-                        .idle => @panic("Invalid state"),
+                        .idle => @panic("Unexpected state .idle in Bun shell 'rm' builtin."),
```
Comment (Jarred-Sumner) [inline src/shell/interpreter.zig:6735]: Nice

**Comment by Jarred-Sumner** [comment]
LGTM but @dylan-conway should look it over incase opinions about __has_install_script I feel like that name was deliberate

**Comment by dylan-conway** [comment]
The field should never be used directly was why it had double underscore, but there's also a comment saying this so LGTM

**Review by dylan-conway — APPROVED** [review]

**Comment by Jarred-Sumner** [comment]
Maybe we should rename it to @“#has_install_scripts”

**Comment by dylan-conway** [comment]
or we could keep the name `has_install_script` and put `get/set` functions on the enum

**Pull request merged**